models/threshold.go

```diff
@@ -0,0 +1,99 @@
+package models
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"github.com/prometheus/common/model"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/models/threshold"
+)
+
+
+type Thresholds []Threshold
+// Customize Thresholds error rates
+type Threshold struct {
+	Percent int   `json:"percent"`
+	Rule    string    `json:"rule"`
+	Alert   string    `json:"alert"`
```

Comment:
I don't think "alert" is good terminology, it has a different meaning.
If it's related to the "condition" of "properties" of the "ThresholdHealth" that would be understood better.